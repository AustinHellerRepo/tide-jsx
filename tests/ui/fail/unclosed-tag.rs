fn main() {
    tide_jsx::html! {
      <ul>
        <li>
      </ul>
    };
}
