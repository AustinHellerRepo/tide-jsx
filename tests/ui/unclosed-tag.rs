fn main() {
    render::html! {
      <ul>
        <li>
      </ul>
    };
}
