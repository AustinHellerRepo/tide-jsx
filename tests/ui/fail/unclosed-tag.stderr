error: Expected closing tag for: <li>
 --> tests/ui/fail/unclosed-tag.rs:5:9
  |
5 |       </ul>
  |         ^^
